tokio-util/src/task/spawn_pinned.rs

```diff
@@ -9,7 +9,57 @@ use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
 use tokio::sync::oneshot;
 use tokio::task::{spawn_local, JoinHandle, LocalSet};
 
-/// A handle to a local pool, used for spawning `!Send` tasks.
+#[derive(Debug)]
+pub struct WorkerIdxError(usize, usize);
```

Comment:
This struct is public, but not actually reachable since the `spawn_pinned` module is not public.